### PR TITLE
Pin cli-table dependency to v0.3.6

### PR DESCRIPTION
**Pull request by phess101**
### Description

Pinning cli-table dependency to v0.3.6, as their v0.3.7 release seems to have introduced a bug with constructing a new table with no arguments https://github.com/Automattic/cli-table/issues/153

### CODE REVIEW

**Comment by phess101** [comment]
I've accepted the CLA, please re-run

**Comment by bkendall** [comment]
Please fix the test issues. Likely, you'll need to do `npm i --save cli-table@0.3.6` and _then_ remove the `^`.

File: package.json [hunk 1]
```diff
@@ -214,4 +214,4 @@
     "typescript": "^3.9.5",
     "typescript-json-schema": "^0.50.1"
   }
-}
+}
```
Comment (bkendall) [inline package.json:217]: Please keep the newline at the end of the file as well.

Comment (phess101) [reply package.json]: Done. Sorry about that, auto save formatting settings on vscode

**Comment by phess101** [comment]
@bkendall updated! Please re-run the tests

**Review by bkendall — APPROVED** [review]

**Comment by firephreek** [comment]
Noticed the integration checks were skipped on this. Maybe they shouldn't so that this issue would be caught in the future? Not being able to run emulators is kind of a big deal :-/

**Pull request merged**